### PR TITLE
Fix #1886

### LINKED ISSUE #1886: Removed transactions are displayed in 1x1 and group chat as pending/failed when closing Sign Transaction screen [0.9.11]
### Description

*Type*: Bug

*Summary*: In release candidate `0.9.11` when I remove transaction on Sign transaction screen and close the screen I can see this transaction is shown in 1x1 chat as `failed` and `pending` in group chat

#### Expected behavior
Transaction is not shown when I removed it and did not sign

#### Actual behavior

1x1 chat
image

Group chat
image

### Reproduction

- Open Status
- Add somebody as contact
- Send txn in 1x1 chat / group chat
- Delete this transaction on Sign screen
- Pay attention to the chat

### Additional Information
* Status version: release `0.9.11` 0.9.10-103-g4672df2b+ (1660) 
* Operating System: Android, iOS
* TF: https://app.testfairy.com/projects/4803622-status/builds/6583606/sessions/11/?accessToken=3zuhuKI2ABGFcvXMxEDN8jwtlsc (from 0:42)

### PR DESCRIPTION
**Pull request by janherich**
There were multiple problems with previous approach of always
expecting explicit "sent" signal from async handler, otherwise
displaying the `/send` message as pending, the most serious
problem was that the command handler is only called once during
send time, and it's problematic to depend on the result of that
on the recepient side, which could be for example offline.

The safer approach is to get rid of status code dependent display
at all and just not send the message for further processing if async
handler returns result code other then `success`.
The code is cleaner as well.
Handler-data path is actually not used anymore, but I decide not to 
delete it for now, as I think it could be beneficial in future for different
kind of handlers then simple `/send` command.

fixes #1886

### Steps to test:
- Open Status
- Open 1-1 chat and send transaction
- Check the messages on the sender/receiver side

[comment]: # (PRs will only be accepted if squashed into single commit.)
status: ready

### CODE REVIEW

**Review by rasom — APPROVED** [review]

**Comment by annadanchenko** [comment]
on sender side: for removed transaction there is message “transaction failed”, for successful transaction - no messages. 
On recipient side: no message for both successful and removed transaction. Expect to get “failed transaction” for both sides. 
screenshot_20170920-090939

img_0617

Happens in 1-1 and in group chat, checked Android and iOS

**Comment by yenda** [comment]
can you fix your indentation please it makes too much noise in the diff

```
  (add-hook 'clojure-mode-hook
            (lambda ()
              (put-clojure-indent 'letsubs 1)
              (put-clojure-indent 'create-class 0)
              (put-clojure-indent 'register-handler-db 0)
              (put-clojure-indent 'register-handler-fx 0)
              (put-clojure-indent 'register-handler 0)
              (put-clojure-indent 'reg-fx 0)
              (put-clojure-indent 'reg-sub 1)
              (put-clojure-indent 'allowed-keys 0)
              (put-clojure-indent 'start 0)
              (put-clojure-indent 'list-item 0)
              (put-clojure-indent 'setTimeout 0)))
```

File: src/status_im/chat/events/commands.cljs [hunk 1]
```diff
@@ -33,91 +34,93 @@
...
- :execute-command-immediately
- [trim-v]
- (fn [_ [{command-name :name :as command}]]
-   (case (keyword command-name)
-     :grant-permissions
-     {:dispatch [:request-permissions
-                 [:read-external-storage]
-                 #(dispatch [:initialize-geth])]}
-     (log/debug "ignoring command: " command))))
+  :execute-command-immediately
```
Comment (yenda) [inline src/status_im/chat/events/commands.cljs:99]: fix indent please

**Review by yenda — CHANGES_REQUESTED** [review]

**Review by yenda — APPROVED** [review]

**Comment by annadanchenko** [comment]
On new build with re-worked send there is new issue: #1903

**Pull request merged**